### PR TITLE
Release

### PR DESCRIPTION
**Pull request by jmzwar**
* Init Dune client & implement chart TVL V3

* Implement chart TVL SNX & chart SNX Supply

* Refactor format TVL Protocols data with reduce

* Implement chart Daily Delegations

* Refactor from Dune to API

* Revert port

* Change API url from http to https

* Fix Header  & rename some of the labels both on the graphs and in the tooltips

* Refactor text

* Change legend SNX to snxUSD

* Change description Mint & Burn chart

* Add padding bottom for V3 Stats page

* Resolved conflicts

* Fix build CI/CD failed

* Fix dedupe lib

* Fix build CI/CD [check]

### CODE REVIEW

**Comment by duyentb95** [comment]
@jmzwar Hey sir! Can you help me fix this code?
`{['/', '/trades', '/actions', '/markets', '/positions, /v3'].includes(
            location.pathname
          )` to `{['/', '/trades', '/actions', '/markets', '/positions'].includes(
            location.pathname
          )` at Header.tsx

Screen Shot 2023-10-23 at 22 58 36

File: v2/perps-v2/ui/package.json [hunk 1]
```diff
@@ -26,13 +26,15 @@
...
     "@synthetixio/wei": "^2.74.4",
+    "axios": "^1.3.3",
     "date-fns": "^2.29.3",
     "ethers": "^5.7.2",
     "framer-motion": "^7.0.0",
     "graphql": "^16.6.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.41.5",
+    "react-query": "^3.39.3",
```
Comment (bachstatter) [inline v2/perps-v2/ui/package.json:37]: This is wrong, we should be using `@tanstack/react-query` which is the newer version

Comment (bachstatter) [reply v2/perps-v2/ui/package.json]: Fixed

**Review by bachstatter — APPROVED** [review]

**Comment by duyentb95** [comment]
@jmzwar @bachstatter Thanks for your help! I'm very sorry for this carelessness and hope you can help me remove '/stats-v3' from the mobile menu? This is old code during testing with Dune and does not have a UI design yet.

Screen Shot 2023-10-24 at 06 27 31

**Pull request merged**